docs/src/components/Button/styles.module.scss

```diff
@@ -0,0 +1,164 @@
+$danger-text: #ef404a;
+$success-text: #59a36c;
+$purple: #7A77FF;
+$white: #FFF;
+$purple-text: #665DE8;
+$cera-pro-font: "Cera Pro", "Avenir Next", Avenir, "Helvetica Neue", Helvetica,
+      Ubuntu, "DejaVu Sans", Arial, sans-serif;
```

Comment:
Can we move these to https://github.com/cube-js/cube.js/blob/master/docs/static/styles/variables.scss